src/Administration/Resources/app/administration/src/app/service/product-stream-condition.service.js

```diff
@@ -6,7 +6,7 @@ const utils = Shopware.Utils;
 
 /**
  * @private
- * @package services-settings
+ * @sw-package framework
```

Comment:
Is this about "product steams" or "dynamic product groups" (as they are called in the admin)? Did we forgot that component in our domain assignments?

Reply:
Good point 🙈 we have not thought of them ... what do you think, should be the assigned domain? inventory or discovery?

Reply:
Good question. The feature in my POV is mostly about building SQL queries to fetch a specific group of products with a fancy admin UI. So it's more about preparing data that can later be used in the storefront and is also closely coupled to the fields available on the product entity.

With that in mind I would suggest inventory as they also maintain the product entity.

@vienthuong what do you think?

Reply:
product stream should belong to inventory, yes :)

Reply:
@vienthuong can you please update our internal handbook to include that component under the inventory domain 🙂